app/templates/introduction.html

```diff
@@ -42,7 +42,7 @@ <h2 class="saturn col-10@l">{{legal_text}}</h2>
           <dl class="dl">
             <dt class="dl__title mercury">PERIOD</dt>
             <dd class="dl__data mars">
-              <span>{{ meta.survey.start_date|format_date }} to {{ meta.survey.end_date|format_date}}
+              <span>{{ meta.survey.start_date|format_date }}{% if meta.survey.end_date %} to {{ meta.survey.end_date|format_date}}{% endif %}
```

Comment:
@benjefferies raises a good point below about it being unit testable as a jinja filter, I think we should do as he suggests. It's also worth looking at the `format_str_as_date_range` filter that is used to generate a from/to date range for the summary screen to see if it could be used.